src/api.ts

```diff
@@ -18,7 +15,7 @@ export const createFetch = (token: string, repo: string) => (query: string) => {
     body: JSON.stringify({
```

Comment:
If left as an Object, `httpie` will cast your body to JSON string for you :) https://github.com/lukeed/polkadot/blob/master/examples/with-graphql/index.js#L28

Reply:
TIL: That's quite neat :+1:

Reply:
It's because I'm lazy